EpiCommonUtils/packages.config

```diff
@@ -12,10 +12,10 @@
   <package id="JetBrains.Annotations" version="2018.3.0" targetFramework="net471" />
   <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net471" />
   <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net471" />
-  <package id="Microsoft.AspNet.WebApi" version="5.2.7" targetFramework="net471" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net471" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.7" targetFramework="net471" />
-  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.7" targetFramework="net471" />
+  <package id="Microsoft.AspNet.WebApi" version="5.2.6" allowedVersions="[5.2,5.2.7)" targetFramework="net471" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.6" allowedVersions="[5.2,5.2.7)" targetFramework="net471" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.6" allowedVersions="[5.2,5.2.7)" targetFramework="net471" />
+  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.6" allowedVersions="[5.2,5.2.7)" targetFramework="net471" />
```

Comment:
Theoretically, not changing major version should maintain backward compatibility, so we should be able to use [5.2, 6), not preventing from upgrading from future non-breaking changes